templates/topic/reports/a11y-1.json

```diff
@@ -0,0 +1,689 @@
+  ],
+  "dct:date": "2024-10-03",
+  "earl:subject": {
+    "@id": "_:subject",
+    "dct:references": [
+      "https://wet-boew.github.io/GCWeb/templates/topic/topic-en.html",
+      "https://wet-boew.github.io/GCWeb/templates/topic/topic-stackedheader-en.html"
+    ],
+    "@type": [
+      "earl:TestSubject",
+      "schema:WebPage"
+    ],
+    "dct:description": "Evaluation of the GC topic template in order to determine if it is aligned with our design and is compliant to our accessibility guideline when used as is without any special customization.",
+    "oa:hasState": [
+      {
+        "@type": "acr:MediaQueryList",
+        "acr:stateValue": "(min-width: 992px) and (max-width: 1199px)",
+        "_:iterator": 0
+      }
+    ],
+    "earl:pointer": {
+      "@type": "oa:CssSelector",
+      "@value": "main"
+    }
+  },
+  "dct:description": "Evaluation of the GC topic template in order to determine if it is aligned with our design and is compliant to our accessibility guideline when used as is without any special customization. The pages were assessed to ensure that they meet the Web Content Accessibility Guideline by manually testing each success criteria.",
+  "earl:result": [
+    {
+      "earl:test": "WCAG21:non-text-content",
+      "earl:subject": {
+        "@id": "_non-text-content_itm1",
+        "dct:isPartOf": "_:subject",
+        "@type": ["earl:TestSubject", "schema:WebPageElement"],
+        "dct:description": "Non-text content includes text alternative that serves the equivalent purpose."
+      },
+      "earl:result": [
+        {
+          "earl:test": "act:rules/wcag2x/sc/1_1_1_situation_a",
+          "earl:outcome": "earl:passed",
+          "earl:mode": "earl:manual",
+          "earl:subject": "_non-text-content_itm1",
+          "dct:description": "Image includes an alt tag to describe the image content."
+        }
+      ],
+      "acr:asset": [],
+      "earl:mode": "earl:manual",
+      "acr:severity": "acr:forEvaluation",
+      "acr:relevancy": "acr:forEvaluation",
+      "dct:description": "",
+      "earl:outcome": "earl:passed",
+      "selected": "act:rulesets/wcag2x/sc/non-text-content#situations"
+    },
+    {
+      "earl:test": "WCAG21:audio-only-and-video-only-prerecorded",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:manual",
+      "acr:severity": "acr:notRelevant",
+      "acr:relevancy": "acr:notRelevant",
+      "dct:description": "",
+      "earl:outcome": "earl:inapplicable",
+      "selected": "act:rulesets/wcag2x/sc/non-text-content#situations"
+    },
+    {
+      "earl:test": "WCAG21:captions-prerecorded",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [],
+      "earl:mode": "earl:manual",
+      "acr:severity": "acr:notRelevant",
+      "acr:relevancy": "acr:notRelevant",
+      "dct:description": "",
+      "earl:outcome": "earl:inapplicable",
+      "selected": ""
```

Comment:
Left over from my tool, please remove

Please apply the same change to all the `earl:test` item.

```suggestion
```

Reply:
Comments addressed